ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java

```diff
@@ -59,6 +59,6 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private boolean isNodeSupportGoInto(Object node) {
-        return node != null && node instanceof Node && ((Node)node).getParent() != null && ((Node)node).supportGoInto();
+        return node != null && node instanceof Node && ((Node)node).supportGoInto();
```

Comment:
But  it's has meaning when you works with few projects and you want to switch and work with one of them. By the way, Eclipse support go into for projects...